### PR TITLE
Release/march release

### PR DESCRIPTION
**Pull request by JeanRoca**
The latest build of our GA website is the one from the branch release/schema-1.5-website. In this PR I have cherry picked the latest GA changes and left out preview features such as 1.6 carousel, viva connections dark mode host app, and berlin containers.

Cherry picked all website changes in including:
- All a11y fixes made by @PankajBhojwani and @carlos-zamora 
- Latest blog posts
- Federated search host app
- Device emulator for designer
- Roadmap link fix
- Latest samples with a11y improvements

Fixed merge conflicts with
- In federated search PR I had to remove berlin containers from adaptivecards-designer.ts
- Same for device emulator PR
- Had a merge conflict with PR #7098 (did not know if this PR was optional) but fixed it by accepting incoming change on line 8 of package.json

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7186)

### CODE REVIEW

**Comment by PankajBhojwani** [comment]
I don't see #7100 in the list, did that get added earlier?

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts [hunk 1]
```diff
@@ -10,9 +11,18 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections (Preview)", "containers/viva-connections-container.css"),
```
Comment (licanhua) [inline source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:13]: Is these change expected?

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts [hunk 2]
```diff
@@ -10,9 +11,18 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
...
+	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
 	new containers.DarkCortanaContainer("Cortana Skills - Dark", "containers/cortana-container-dark.css"),
 	new containers.LightCortanaContainer("Cortana Skills - Light", "containers/cortana-container-light.css"),
 	new containers.BotFrameworkContainer("Bot Framework Other Channels (Image render)", "containers/bf-image-container.css"),
 	new containers.CortanaClassicContainer("Cortana Skills (Classic)", "containers/cortana-classic-container.css"),
-]
+	new containers.FederatedSearchContainer("Federated Search","containers/federated-search-container.css"),
```
Comment (licanhua) [inline source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:19]: Do you expect this change?

**Comment by JeanRoca** [comment]
> I don't see #7100 in the list, did that get added earlier?
> 
> EDIT: also #7102

The changes are there, but since I had merge conflicts they did not maintain the same descriptions. #7100 is in commit https://github.com/microsoft/AdaptiveCards/pull/7186/commits/9990befe6d40592b68fb6861f8617fd6ad437c0f

And #7102 is in commit https://github.com/microsoft/AdaptiveCards/pull/7186/commits/87c41bc35f0e08f24519f00acbe8714be8afe875

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts (hunk 1 above)
Comment (JeanRoca) [reply source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:13]: Yes viva connections is no longer in preview. The line below adds the same line without preview in name.

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts (hunk 2 above)
Comment (JeanRoca) [reply source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:19]: Yes the federated search host app is included in this release.

File: source/nodejs/aclint/src/index.ts [hunk 3]
```diff
@@ -1,256 +0,0 @@
-#!/usr/bin/env node
-
-import * as yargs from 'yargs';
```
Comment (licanhua) [inline source/nodejs/aclint/src/index.ts:3]: Don't bring clang-format, js-ci, removing aclint, other changes into this PR

File: source/nodejs/adaptivecards-designer/package.json [hunk 4]
```diff
@@ -41,25 +41,15 @@
...
-		"monaco-editor": "^0.20.0"
+		"monaco-editor": "^0.29.1"
 	},
 	"devDependencies": {
-		"@typescript-eslint/eslint-plugin": "^3.4.0",
-		"@typescript-eslint/parser": "^3.4.0",
 		"adaptive-expressions": "^4.11.0",
-		"adaptivecards": "^2.10.0",
-		"adaptivecards-templating": "^2.2.0",
+		"adaptivecards": "^3.0.0-beta.1",
```
Comment (licanhua) [inline source/nodejs/adaptivecards-designer/package.json:48]: We don't need this change and it should stay in 2.10.x

Comment (JeanRoca) [reply source/nodejs/adaptivecards-designer/package.json]: fixed by reverting those commits

File: source/nodejs/aclint/src/index.ts (hunk 3 above)
Comment (JeanRoca) [reply source/nodejs/aclint/src/index.ts]: reverted thanks!

**Comment by JCMartell** [comment]
I don't see #7096 in the changes here. Any reason for not including it?

**Comment by JeanRoca** [comment]
> I don't see #7096 in the changes here. Any reason for not including it?

Hey @JCMartell! This is a website release, not a schema release. We release schema changes to the website once we have an sdk release.

**Review by licanhua — APPROVED** [review]

**Review by almedina-ms — APPROVED** [review]

**Pull request merged**